ethereum/contracts/Bridge.sol

```diff
@@ -37,7 +37,7 @@ contract Bridge {
 
         bytes memory payload = _message.slice(32, _message.length-1);
         Application app = Application(appID);
-        app.handle(payload);
+        app.submit(payload);
```

Comment:
It should be `app.handle()` to mirror the substrate naming conventions. Following this language: The relayer _submits_ messages to the bridge, and the bridge asks an application to _handle_ a message.

I realize my Solidity changes made this confusing, they were stop gap measures which made applications look like a bridge.

Reply:
agreed, this makes much more sense.

Reply:
https://github.com/Snowfork/polkadot-ethereum/pull/75/commits/3ce50f6763d7cd67b4a9968a76b0de19118fad56